cloud/linode/common.go

```diff
@@ -4,11 +4,16 @@ import (
 	"fmt"
 	"strconv"
 	"strings"
+	"time"
 
 	"github.com/linode/linodego"
 )
 
-const providerIDPrefix = "linode://"
+const (
+	providerIDPrefix = "linode://"
+	// context timeout for API requests
+	requestTimeout = 120 * time.Second
```

Comment:
Should I make this configurable? If yes, which option should be better:
1. Adding new flag which can be specified when running the image.
2. Adding an environment variable which when set configures the requestTimeout.